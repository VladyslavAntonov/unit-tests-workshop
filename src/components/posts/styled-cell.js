import { TableCell } from '@material-ui/core';
import { withStyles } from '@material-ui/core/styles';

export const StyledCell = withStyles((theme) => ({
  head: {
    backgroundColor: theme.palette.info.light,
    color: theme.palette.common.black
  },
  body: {
    fontSize: 12
  }
}))(TableCell);
