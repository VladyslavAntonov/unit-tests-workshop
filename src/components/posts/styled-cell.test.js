import { shallow } from 'enzyme';

import { StyledCell } from './styled-cell';

describe('StyledCell', () => {
  let component;

  beforeEach(() => {
    component = shallow(<StyledCell>content</StyledCell>);
  });

  it('matches the snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
