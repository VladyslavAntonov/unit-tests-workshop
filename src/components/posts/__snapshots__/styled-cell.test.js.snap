// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledCell matches the snapshot 1`] = `
<WithStyles(ForwardRef(TableCell))
  classes={
    Object {
      "body": "WithStyles(ForwardRef(TableCell))-body-2",
      "head": "WithStyles(ForwardRef(TableCell))-head-1",
    }
  }
>
  content
</WithStyles(ForwardRef(TableCell))>
`;
